tests/ttnn/unit_tests/operations/test_reduce_scatter_nightly.py

```diff
@@ -106,21 +101,20 @@ def test_reduce_scatter_nightly(
         ([1, 8, 1024, 1024], 3, ttnn.TILE_LAYOUT),
         ([1, 4, 1024, 1024], 3, ttnn.TILE_LAYOUT),
         ([1, 4, 2048, 1024], 3, ttnn.TILE_LAYOUT),
-        ([1, 1, 32, 32], 3, ttnn.TILE_LAYOUT),
-        ([1, 1, 32, 64], 3, ttnn.TILE_LAYOUT),
-        ([1, 1, 64, 64], 3, ttnn.TILE_LAYOUT),
-        ([1, 1, 32, 128], 3, ttnn.TILE_LAYOUT),
-        ([1, 1, 32, 256], 3, ttnn.TILE_LAYOUT),
         ([1, 1, 32, 512], 3, ttnn.TILE_LAYOUT),
         ([1, 1, 32, 1024], 3, ttnn.TILE_LAYOUT),
         ([1, 1, 32, 2048], 3, ttnn.TILE_LAYOUT),
         ([1, 1, 128, 1024], 3, ttnn.TILE_LAYOUT),
-        # Has worker slice size warning - defaults to 1x1
         ([1, 1, 128, 8192], 3, ttnn.TILE_LAYOUT),
-        # Always fails with bfp8_b
         ([1, 1, 2048, 1024], 3, ttnn.TILE_LAYOUT),
-        # Has worker slice size warning - defaults to 1x1
         ([1, 1, 2048, 8192], 3, ttnn.TILE_LAYOUT),
+        # These shapes result in some workers with no work, which is currently
```

Comment:
was this working before?

Reply:
A long time ago. It looks like these tests, for whatever reason, have just been getting skipped in nightly CI for a little while so the regression was never caught. I'm guessing they may have regressed when I added initial line reduce scatter a few weeks ago because it was a major refactor.

Unfortunately it seems it's not enough to just check the green/red status but I'll need to also periodically look closely at the log even for green pipelines to make sure all tests are still running...